scripts/opentargets-disease-meddra-word2vec.sc

```diff
@@ -326,30 +340,32 @@ object ETL extends LazyLogging {
       .orderBy($"traitKey".asc)
       .persist(StorageLevel.DISK_ONLY)
 
-    DN.write.json(s"${output}/DiseaseLabels")
-    MN.write.json(s"${output}/TraitLabels")
-
-    val terms = DN.selectExpr("efoKey as terms")
-      .unionByName(MN.selectExpr("traitKey as terms")).persist()
-
-    val w2v = new Word2Vec()
-      .setWindowSize(5)
-      .setVectorSize(100)
-      .setNumPartitions(16)
-      .setMaxIter(3)
-      .setMinCount(0)
-      .setStepSize(0.025)
-      .setInputCol("terms")
-      .setOutputCol("predictions")
-
-    val w2vModel = w2v.fit(terms)
-
-    w2vModel.save(s"${output}/Model")
-    w2vModel.getVectors
-      .withColumn("vector", vector_to_array($"vector"))
-      .write.json(s"${output}/ModelVectors")
+//    DN.write.json(s"${output}/DiseaseLabels")
```

Comment:
Let's delete the commented lines 297, 299-301 and 343-366.

Reply:
it needs to stand it up there like that because there are two ways of loading the models. one is internal and the other is external. I will separate those in further commits